src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/ErrorMessage.java

```diff
@@ -37,10 +38,16 @@ public ErrorMessage(Exception exception, int status) {
 
     private String fetchType() { return exception.getClass().getSimpleName(); }
 
-    private String fetchReason() { return emptyStringIfNull(exception.getLocalizedMessage()); }
+    private String fetchReason() {
+        return status == RestStatus.BAD_REQUEST.getStatus()
+                ? "SQL in the query request is not valid"
```

Comment:
np: `"Invalid SQL query"`

Reply:
How about other client-side errors? Should we at least print out a generic message for everything else which is 4xx and then another one for 5xx?

Reply:
Done!